src/ProtonPlayer.js

```diff
@@ -3,24 +3,28 @@ import Bowser from 'bowser';
 import ProtonPlayerError from './ProtonPlayerError';
 import { debug } from './utils/logger';
 import getContext from './getContext';
-import noop from './utils/noop';
 import initializeiOSAudioEngine from './utils/initializeiOSAudioEngine';
 import Player from './Player';
 import Cursor from './Cursor';
+import EventEmitter from './EventEmitter';
 
 initializeiOSAudioEngine();
 
-export default class ProtonPlayer {
-  constructor({
-    onReady = noop,
-    onError = noop,
-    onPlaybackProgress = noop,
-    onPlaybackEnded = noop,
-    onTrackChanged = noop,
-    volume = 1,
-  }) {
+const PlaybackState = {
```

Comment:
Do you think it'd be worth exposing this? Maybe the "client" code case make use of these constants in order to prevent typos. If so, these could be used in the example pages too.

Reply:
They are exposed via the static `ProtonPlayer.PlaybackState` property.